packages/analytics-browser/test/config.test.ts

```diff
@@ -241,6 +241,23 @@ describe('config', () => {
       );
       expect(config.cookieOptions?.domain).toEqual('amplitude.com');
     });
+
+    test('should use trackingOptions', async () => {
+      const config = await Config.useBrowserConfig(
+        apiKey,
+        {
+          trackingOptions: {
+            ipAddress: false,
+            language: false,
+            platform: false,
+          },
+        },
+        new AmplitudeBrowser(),
+      );
+      expect(config.trackingOptions.ipAddress).toEqual(false);
+      expect(config.trackingOptions.language).toEqual(false);
+      expect(config.trackingOptions.platform).toEqual(false);
+    });
```

Comment:
we can also test the possible values

```suggestion
    test.each([
      [true, true]
      [undefined, true]
      [false, false]
    ])('should use trackingOptions', async (input, expected) => {
      const config = await Config.useBrowserConfig(
        apiKey,
        {
          trackingOptions: {
            ipAddress: input,
            language: input,
            platform: input,
          },
        },
        new AmplitudeBrowser(),
      );
      expect(config.trackingOptions.ipAddress).toEqual(expected);
      expect(config.trackingOptions.language).toEqual(expected);
      expect(config.trackingOptions.platform).toEqual(expected);
    });
```